spanner/cloud-client/src/test/java/com/example/spanner/SpannerSampleIT.java

```diff
@@ -241,6 +252,83 @@ public void testSample() throws Exception {
     assertThat(out).contains("1 1 Total Junk");
     out = runSample("querywithqueryoptions");
     assertThat(out).contains("1 1 Total Junk");
+
+    String backupName =
+        String.format(
+            "%s_%02d",
+            dbId.getDatabase(), LocalDate.now().get(ChronoField.ALIGNED_WEEK_OF_YEAR));
+    BackupId backupId = BackupId.of(dbId.getInstanceId(), backupName);
+
+    out = runSample("createbackup");
+    assertThat(out).contains("Created backup [" + backupId + "]");
+
+    out = runSample("cancelcreatebackup");
+    assertThat(out).contains(
+        "Backup operation for [" + backupId + "_cancel] successfully cancelled");
+
+    out = runSample("listbackupoperations");
+    assertThat(out).contains(
+        String.format(
+            "Backup %s on database %s pending:",
+            backupId.getName(),
+            dbId.getName()));
+
+    out = runSample("listbackups");
+    assertThat(out).contains(databaseId);
```

Comment:
I think it would be good to test at least some of outputs with various filters. Let me know if you think it would be too flaky.

Reply:
With the new filters, the output should now be predictable, so I've added checks to verify that the backup that was created is returned by all list operations.